### ISSUE TITLE
Improve Web Interface

### ISSUE THREAD

**Issue by amarjeetkapoor1** [GreatDevelopers/Sim#8]
Improve Web Interface.
- [ ] Create Main page Web interface of Sim for all features.
- [ ] Improve URLs of apps in Sim.
- [ ] Improve UI for all apps in Sim.

**Comment by TamandeepSingh** [comment]
I would like to do work with UI and other mentioned issues.

**Comment by mandeeps708** [comment]
Break down the issues. Keep it simple, isolated and rather smaller issue to
make it interesting.

**Comment by TamandeepSingh** [comment]
On Sun, Oct 9, 2016 at 10:09 PM, Mandeep Singh notifications@github.com wrote:

> Break down the issues. Keep it simple, isolated and rather smaller issue to
> make it interesting.
> On Oct 8, 2016 3:02 PM, "amarjeet singh kapoor" notifications@github.com
> wrote:
> 
> > Improve Web Interface.
> > - Create Main page Web interface of Sim for all features.
> > - Improve URLs of apps in Sim.
> > - Improve UI for all apps in Sim.

Have I need to choose one from them?

## 

Regards,
Tamandeep Singh
https://github.com/TamandeepSingh
https://tamandeepsingh.wordpress.com/

"Its the will that make u do"

**Comment by amarjeetkapoor1** [comment]
No, You should send a single pull request as this is just collection of some trivial problems but its not a compulsion. Just solve a problem and commit and mark it done in this issue.

**Comment by TamandeepSingh** [comment]
On Sun, Oct 9, 2016 at 10:36 PM, amarjeet singh kapoor
notifications@github.com wrote:

> No, You should send a single pull request as this is just collection of some trivial problems but its not a compulsion. Just solve a problem and commit and mark it done in this issue.
> 
> Okay.

## 

Regards,
Tamandeep Singh
https://github.com/TamandeepSingh
https://tamandeepsingh.wordpress.com/

"Its the will that make u do"
